setup.py

```diff
@@ -74,7 +74,8 @@ def configuration(parent_package='', top_path=None):
 
 
 if __name__ == '__main__':
-    from numpy.distutils.core import setup, Extension
+    #from numpy.distutils.core import setup, Extension
+    from setuptools import setup, Extension
```

Comment:
we've been using the builds for about 6 months without issues.. I now a lot of changes have come through setuptools over the last couple of years... or configuration may be ignored... doesn't seem to effect the package operation..   having  binary wheels is a really nice on installation time!